rubin_jupyter_utils/lab/notebook/catalog.py

```diff
@@ -1,40 +1,28 @@
 import requests
 import pyvo
 import pyvo.auth.authsession
-from rubin_jupyter_utils.helpers import get_access_token, parse_access_token
+from rubin_jupyter_utils.helpers import get_access_token
 from rubin_jupyter_utils.config import RubinConfig
 
 
+class TokenNotFoundError(Exception):
+    """No access token was found."""
+
+
 def _get_tap_url():
     rc = RubinConfig()
     tapurl = rc.external_tap_url or (rc.external_instance_url +
                                      rc.tap_route)
     return tapurl
 
 
-def _get_gf_analyze_endpoint():
-    rc = RubinConfig()
-    gf = rc.external_gafaelfawr_url or (rc.external_instance_url +
-                                        rc.gafaelfawr_route)
-    gfendpoint = gf + '/analyze'
-    return gfendpoint
-
-
-def _get_token():
-    """Returns access token if (and only if) it is valid; otherwise throws
-    exception."""
-    token = get_access_token()
-    gfendpoint = _get_gf_analyze_endpoint()
-    # parse_access_token() will throw an exception if the token is not
-    #  valid.
-    parse_access_token(endpoint=gfendpoint, token=token)
-    return token
-
-
 def _get_auth():
     tap_url = _get_tap_url()
     s = requests.Session()
-    s.headers["Authorization"] = "Bearer " + _get_token()
+    tok = get_access_token()
+    if not tok:
+        raise TokenNotFoundError
```

Comment:
This loses the message explaining where we looked for the token. I think the exception should be thrown by the code in rubin-jupyter-helpers in this way of constructing the code so that it can provide the helpful message.